### PR TITLE
Add Symfony 7 and PHP 8.2 support

### PR DESCRIPTION
**Pull request by dfridrich**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -1,3 +1,4 @@
 parameters:
     checkMissingIterableValueType: false
     ignoreErrors:
+    checkGenericClassInNonGenericObjectType: false
```
Comment (dfridrich) [inline phpstan.neon:4]: @mhujer this is to discuss, whether to disable this check or not :-)

Comment (mhujer) [reply phpstan.neon:4]: @dfridrich let's disable it

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -23,6 +23,11 @@ jobs:
                         operating-system: windows-latest
                         coverage: xdebug
 
+                    -   php-versions: ['8.2', '8.3']
```
Comment (mhujer) [inline .github/workflows/main.yml]: I'm not sure whether this will work as array

Comment (dfridrich) [reply .github/workflows/main.yml]: Nope :-) Fixed in new commit

**Comment by mhujer** [comment]
@dfridrich not all SF packages installed during build are v7 because installed version of `matthiasnoback/symfony-dependency-injection-test` does not support it.
image

I don't know how much effort it would be to upgrade it to latest which supports v7

**Comment by dfridrich** [comment]
@mhujer I will play with it yet, anyway, could you allow auto CI without your approval? It would be much more comfy 🙏

**Comment by mhujer** [comment]
@dfridrich I sent you an invite to the project

**Comment by mhujer** [comment]
@dfridrich Thank you! Merged and new release was tagged (I really appreciate that you also adjusted the README so all I had to do was merging and tagging 👍 )

**Pull request merged**